backends/gdx-backend-lwjgl/src/com/badlogic/gdx/backends/lwjgl/LwjglGraphicsOffScreen.java

```diff
@@ -0,0 +1,414 @@
+/*******************************************************************************
+ * Copyright 2011 See AUTHORS file.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+
+package com.badlogic.gdx.backends.lwjgl;
+
+import java.awt.*;
+
+import org.lwjgl.*;
+import org.lwjgl.opengl.*;
+
+import com.badlogic.gdx.*;
+import com.badlogic.gdx.graphics.*;
+import com.badlogic.gdx.graphics.Cursor.*;
+import com.badlogic.gdx.graphics.GL20;
+import com.badlogic.gdx.graphics.GL30;
+import com.badlogic.gdx.graphics.glutils.*;
+import com.badlogic.gdx.utils.*;
+
+/** LwjglGraphicsOffScreen implementation derived from {@link LwjglGraphics}. This class uses an external GL context.
+ * 
+ * @author ClassX */
+public class LwjglGraphicsOffScreen extends AbstractGraphics {
```

Comment:
Is this class strictly necessary?
It seems to be pretty application specific, i.e., what does it offer to the average developer?

Reply:
It looks like it's enabled by making multi-context possible, so it seems to make sense here. As far as I can tell, this would be useful for off-screen framebuffer rendering when you have many framebuffers at once. I haven't read this large PR thoroughly, though.

Reply:
yep, as Tommy writes, not every application is starting with a Display.
There may be cases where you need to create only offscreen contexts using FBOs. This is expecially true for Swing apps,